frontend/src/pages/protocol-token/index.vue

```diff
@@ -6,59 +6,81 @@
 			<div
 				class="flex flex-row justify-center lg:place-content-end mb-4 lg:mb-0"
 			></div>
-			<!-- <header class="flex justify-between items-center mb-4">
+			<header class="flex justify-between items-center mb-4">
 				<h1 class="text-xl">Overview</h1>
 			</header>
 			<FtbCarousel non-carousel-classes="grid-cols-4">
 				<CarouselSlide class="w-full lg:h-full">
-					<KeyValueChartCard class="w-96 lg:w-full" :y-values="[[null]]" :is-loading="false">
-						<template #title>Total Market Cap</template>
+					<KeyValueChartCard
+						class="w-96 lg:w-full"
+						:y-values="[[null]]"
+						:is-loading="false"
+					>
+						<template #title>Total Token Supply</template>
 						<template #value>
 							<p class="font-bold text-2xl mt-2">
 								{{
-									
 									numberFormatter(
-										pltEventMetricsDataRef?.pltEventMetrics.lastCumulativeTotalSupply
+										pltTokenDataRef.reduce(
+											(acc, coin) =>
+												acc +
+												(coin?.totalSupply ?? 0) /
+													Math.pow(10, Number(coin?.decimal)),
```

Comment:
<PLTAmount> component or a common helper function so we don't re-write the same math formula.